.evergreen/scripts/compile-libmongocrypt.sh

```diff
@@ -5,16 +5,7 @@ compile_libmongocrypt() {
   declare -r mongoc_dir="${2:?}"
   declare -r install_dir="${3:?}"
 
-  git clone -q --depth=1 https://github.com/mongodb/libmongocrypt --branch 1.7.0 || return
-
-  # TODO: remove once latest libmongocrypt release contains commit c6f65fe6.
-  {
-    pushd libmongocrypt || return
-    echo "1.7.0+c6f65fe6" >|VERSION_CURRENT
-    git fetch -q origin master || return
-    git checkout -q c6f65fe6 || return # Allows -DENABLE_MONGOC=OFF.
-    popd || return                     # libmongocrypt
-  }
+  git clone -q --depth=1 https://github.com/mongodb/libmongocrypt --branch 1.8.0-alpha0 || return
```

Comment:
Suggest updating all other instances of libmongocrypt cloning in Evergreen scripts for consistency.

Reply:
Whoops, I had forgotten I refactored the instances into this script for this very reason. 😅